scripts/install.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -euo pipefail
+
+LOG_START='\n\e[1;36m'  # new line + bold + cyan
+LOG_END='\n\e[0m'       # new line + reset
+DONE_START='\n\e[1;32m' # new line + bold + green
+DONE_END='\n\n\e[0m'    # new line + reset
+
+COVERAGE_POOL_PATH=$(realpath $(dirname $0)/../)
+
+# Defaults, can be overwritten by env variables/input parameters
+NETWORK_DEFAULT="local"
```

Comment:
Because of f12c0ec we need to change the default network as well. @nkuba

Reply:
https://github.com/keep-network/coverage-pools/pull/153